### PR TITLE
dev: add admin repair table statement

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is the related PR or file link(s)? 

- [ ] Reference link(s):
- [x] This PR is to align with:pingcap/docs-cn#2205, pingcap/docs-cn#2212
- [ ] N/A (not applicable)

### Which TiDB version(s) does your changes apply to? 

- [ ] All active versions: dev, v3.0, v2.1, v3.1
- [x] dev (the latest development version)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] All active and inactive versions
- [ ] N/A (not applicable)

**Note:** If your changes apply to multiple TiDB versions, make sure you update the documents in the **corresponding** version folders such as "dev", "v3.0", "v2.1" and "v3.1" in this PR.

- [ ] Updated one version first. Will update other versions after I get two LGTMs.

### CODE REVIEW

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 1]
```diff
@@ -66,6 +66,18 @@ The TiDB configuration file supports more options than command-line parameters.
...
+
+- Determines whether to enable the untrusted repair mode. When the `repair-mode` is set to `true`, bad tables in the `repair-table-list` cannot be loaded.
+- Default value: `false`
+- The syntax for the fix is not supported by default, and all table information is loaded when TiDB is started.
+
+### `repair-table-list`
+
+- Lists all bad tables that need to be fixed in instances in the case that the `repair-mode` is set to `true`. An example for the list is: ["db.table1","db.table2"...].
+- Default value: []
+- With the default setting, the list is empty, which means that there is no information about bad tables that need to be fixed.
```
Comment (AilinKid) [inline dev/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
- With the default setting, the list is empty, which means that there is no information about bad tables that need to be repaired.
```

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 2]
```diff
@@ -66,6 +66,18 @@ The TiDB configuration file supports more options than command-line parameters.
...
 
+### `repair-mode`
+
+- Determines whether to enable the untrusted repair mode. When the `repair-mode` is set to `true`, bad tables in the `repair-table-list` cannot be loaded.
+- Default value: `false`
+- The syntax for the fix is not supported by default, and all table information is loaded when TiDB is started.
+
+### `repair-table-list`
+
+- Lists all bad tables that need to be fixed in instances in the case that the `repair-mode` is set to `true`. An example for the list is: ["db.table1","db.table2"...].
```
[the remark this answers is not in the document]
Comment (toutdesuite) [reply dev/reference/configuration/tidb-server/configuration-file.md]: Thanks. The file is updated.

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 3]
```diff
@@ -66,6 +66,18 @@ The TiDB configuration file supports more options than command-line parameters.
 - Determines whether to treat the `utf8` character set in old tables as `utf8mb4`.
 - Default value: `true`
 
+### `repair-mode`
+
+- Determines whether to enable the untrusted repair mode. When the `repair-mode` is set to `true`, bad tables in the `repair-table-list` cannot be loaded.
+- Default value: `false`
+- The syntax for the `repair` is not supported by default, and all table information is loaded when TiDB is started.
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
- The `repair` syntax is not supported by default. This means that all tables are loaded when TiDB is started.
```

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 4]
```diff
@@ -66,6 +66,18 @@ The TiDB configuration file supports more options than command-line parameters.
...
+
+### `repair-table-list`
+
+- Lists all bad tables that need to be repaired in instances in the case that the `repair-mode` is set to `true`. An example for the list is: ["db.table1","db.table2"...].
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
- `repair-table-list` is only valid when [`repair-mode`](#repair-mode) is set to `true`. `repair-table-list` is a list of bad tables that need to be repaired in an instance. An example of the list is: ["db.table1","db.table2"...].
```

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 5]
```diff
@@ -66,6 +66,18 @@ The TiDB configuration file supports more options than command-line parameters.
...
+- Default value: []
+- With the default setting, the list is empty, which means that there is no information about bad tables that need to be repaired.
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
- The list is empty by default. This means that there are no bad tables that need to be repaired.
```

File: dev/reference/sql/statements/admin.md [hunk 6]
```diff
@@ -76,6 +104,7 @@ mysql> admin show ddl jobs;
     > - If the jobs you want to cancel are finished, the cancellation operation fails.
 
 - `ADMIN CHECK TABLE tbl_name [, tbl_name] ...`: To check the consistency of all the data in the specified table and corresponding indexes. If the check is passed, an empty result will be returned. On failure, an error message will indicate that data is inconsistent.
+- `ADMIN REPAIR TABLE tbl_name CREATE TABLE STATEMENT`: In extreme cases, it is used to untrusted overwrite the meta information of the table in the storage layer. Here "untrusted" means that the meta information of the original table must be checked by a real person to ensure the unique source of information: `CREATE TABLE STATEMENT`. To make the statement take effect, you need to set the [`repair-mode`](/dev/reference/configuration/tidb-server/configuration-file.md#repair-mode) in the configuration file to `true`, and make sure that the names of tables that need to be fixed are on the [`repair-table-list`](/dev/reference/configuration/tidb-server/configuration-file.md#repair-table-list).
```
Comment (yikeke) [inline dev/reference/sql/statements/admin.md]:
```suggestion
- `ADMIN REPAIR TABLE tbl_name CREATE TABLE STATEMENT`: In extreme cases, this statement is used to overwrite the metadata of the stored table in an untrusted way. Here "untrusted" means that it must be ensured that the metadata of the original table is covered by the `CREATE TABLE STATEMENT` operation. In order to use this "REPAIR" statement, you need to first enable the [`repair-mode`](/dev/reference/configuration/tidb-server/configuration-file.md#repair-mode) configuration item, and make sure that the tables to be repaired are listed in the [`repair-table-list`](/dev/reference/configuration/tidb-server/configuration-file.md#repair-table-list).
```

**Comment by yikeke** [comment]
I edited a bit to make the content more concise. PTAL at my suggested changes above. Thanks. @AilinKid

**Pull request merged**